src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {

```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line before the class declaration.